test/controllers/AddClosureMetadataControllerSpec.scala

```diff
@@ -547,11 +766,30 @@ class AddClosureMetadataControllerSpec extends FrontEndTestHelper {
     )
     addClosureMetadataPageAsString must include(
       """        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
-        |            Description Public
+        |            Is the description closed?
         |        </legend>""".stripMargin
     )
   }
 
+  private def checkForExpectedClosureMetadataDependenciesFormPageContent(addClosureMetadataPageAsString: String, fullNameAndName: Map[String, String]): Unit = {
```

Comment:
I'm not sure if this is an issue but these checks are very dependant on white space. I had to refactor a html page recently and I spent a lot of time trying to get the whitespace right in the test. You could try something like `addClosureMetadataPageAsString.replaceAll("\\s{2,}", "")` which will remove all spaces apart from single ones which might help. 
I'm not sure that it will though so if this makes things more complicated then ignore me.